integration-cli/docker_cli_kill_test.go

```diff
@@ -33,6 +33,22 @@ func (s *DockerSuite) TestKillContainer(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestKillofStoppedContainer(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "top")
+	out, _, err := runCommandWithOutput(runCmd)
+	c.Assert(err, check.IsNil)
+
+	cleanedContainerID := strings.TrimSpace(out)
+
+	stopCmd := exec.Command(dockerBinary, "stop", cleanedContainerID)
+	out, _, err = runCommandWithOutput(stopCmd)
+	c.Assert(err, check.IsNil, check.Commentf("failed to stop container: %s, %v", out, err))
+
+	killCmd := exec.Command(dockerBinary, "kill", "-s", "30", cleanedContainerID)
+	_, _, err = runCommandWithOutput(killCmd)
+	c.Assert(err, check.Not(check.IsNil), check.Commentf("Kill succeeded on a stopped container"))
```

Comment:
I don't think this addresses the previous comment, you're not checking the output to make sure you get the new error message you're generating.